### DISCUSSION TITLE
Disable rule addition on style change

### DISCUSSION THREAD

**Discussion by gandhar-workik** [GrapesJS/grapesjs#4229] [Q&A]
Hi @artf, when some property is changed in the style manager, a rule object is automatically added(/updated) to(/from) the rule list. I want to stop that from happening. How can I do it?

**Comment by artf** [comment]
Well, that's the whole point of styling, you need that rule to be added/updated 😅

**Reply by gandhar-workik** [reply]
Thanks for the reply. I was trying to add some custom logic on the styleUpdate event and the added/updated rule was causing some inconsistencies due to which I wanted to suppress it
